### PR TITLE
Attack action

### PR DESCRIPTION
**Pull request by apple00juice**
Added the damage Indicator Popup, and did attack actions without API call

### CODE REVIEW

**Review by mkorob — APPROVED** [review]

**Review by hilalcomak — APPROVED** [review]

**Review by luis-tm — APPROVED** [review]

File: src/models/UnitModel.js [hunk 1]
```diff
@@ -213,7 +213,9 @@ class UnitModel {
             }
 
             // After all children have been added sort by f(x)
-            frontier.sort((a, b) => { return a[2] - b[2] });
+            frontier.sort((a, b) => {
```
Comment (paolovic) [inline src/models/UnitModel.js:216]: "return" is redundant

**Review by paolovic — APPROVED** [review]

File: src/models/UnitModel.js (hunk 1 above)
Comment (apple00juice) [reply src/models/UnitModel.js:216]: But sort requires that the function returns a value that it knows which is larger 😕

Comment (paolovic) [reply src/models/UnitModel.js:216]: the arrow indicates "return" ;)

Comment (paolovic) [reply src/models/UnitModel.js:216]: Bildschirmfoto 2022-05-01 um 19 29 47

Comment (paolovic) [reply src/models/UnitModel.js:216]: you could even omit the parenthesis I assume ^^
(a, b) => a[2] - b[2]

**Comment by paolovic** [comment]
great as always :)
tyvm!

File: src/models/UnitModel.js (hunk 1 above)
Comment (apple00juice) [reply src/models/UnitModel.js]: I see 👍

**Pull request merged**